File: Python/1036.py
# BEE 1036 | FÓRMULA DE BHASKARA

import math

line = input().split(' ')
A = float(line[0])
B = float(line[1])
C = float(line[2])

delta = pow(B, 2) - 4 * A * C

if A != 0 and delta > -1:
    R1 = (-B + math.sqrt(delta)) / (2 * A)
    R2 = (-B - math.sqrt(delta)) / (2 * A)
    print("R1 = %0.5f" % R1)
    print("R2 = %0.5f" % R2)
else:
    print("Impossivel calcular")

File: PHP/1036.php
<?php 

# BEE 1036 | FÓRMULA DE BHASKARA

    $data = fscanf(STDIN, "%f %f %f");

    $A = (float) $data[0];
    $B = (float) $data[1];
    $C = (float) $data[2];

    $delta = pow($B, 2) - 4 * $A * $C;

    if($A != 0 && $delta > -1) {
        $R1 = (-$B + sqrt($delta)) / (2 * $A);
        $R2 = (-$B - sqrt($delta)) / (2 * $A);

        echo("R1 = " . number_format($R1, 5, '.', '')) . PHP_EOL;
        echo("R2 = " . number_format($R2, 5, '.', '') . PHP_EOL);
    } else {
        echo("Impossivel calcular" . PHP_EOL);
    }